docs/guidance/rayjob.md

```diff
@@ -0,0 +1,129 @@
+## Ray Job (alpha)
+
+> Note: This is the alpha version of Ray Job Support in KubeRay. There will be ongoing improvements for Ray Job in the future releases.
+
+### Prerequisite
+
+* Ray 1.10 and above.
+* KubeRay v0.3.0 or master
+
+### What is a RayJob?
+
+The RayService is a new custom resource (CR) supported by KubeRay in v0.3.0.
+
+A RayService manages 2 things:
+* RayCluster: Manages resources in kubernetes cluster.
+* Ray Serve Deployment Graph: Manages users' serve deployment graph.
```

Comment:
We should probably link to the Ray docs to explain what a Ray Serve Deployment Graph is.
cc @simon-mo @brucez-anyscale

Reply:
I originally copy some contents and structure from RayService and forget to clean them up. Now it's in good shape